cmd/server-main.go

```diff
@@ -246,13 +246,13 @@ func serverMain(ctx *cli.Context) {
 	var handler http.Handler
 	handler, err = configureServerHandler(globalEndpoints)
 	if err != nil {
-		logger.Fatal(uiErrUnexpectedError(err), "Unable to configure one of server's RPC services.")
+		logger.Fatal(uiErrUnexpectedError(err), "Unable to configure one of server's RPC services")
```

Comment:
why remove the `.` is it removed everywhere?

Reply:
> why remove the . is it removed everywhere?

It is not consistent, I'll remove them in this PR.

Reply:
> why remove the . is it removed everywhere?

Fixed